docs/user-guide/geoprocessing-tool.md

```diff
@@ -5,10 +5,22 @@ This tool aims to provide a set of geo-processing utilities on layers present in
 !!! note
     The **GeoProcessing Tool** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed or directly included in the standard [application configuration](../developer-guide/local-config.md).
 
+!!! warning
+    To avoid errors in geometry overlay methods and therefore correctly use the *GeoProcessing tool*, the [JTS system property](https://github.com/locationtech/jts/blob/master/USING.md#jts-system-properties) must be added to the GeoServer in use (enable the `-Djts.overlay=ng` property).
```

Comment:
```suggestion
    To prevent possible errors in geometry overlay methods due to complex and heavy geometries, the [JTS system property](https://github.com/locationtech/jts/blob/master/USING.md#jts-system-properties) must be added to the GeoServer in use (the `-Djts.overlay=ng` should be enabled in JAVA properties).
```